### PR TITLE
Create Installing Slimefun Page

### PR DESCRIPTION
**Pull request by taydeo**
Just a simple installing Slimefun page, sorry for not doing this earlier.

### CODE REVIEW

File: pages/installing-slimefun.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+**Researches.yml** allows you to edit the XP Values of items in Slimefun, as well as their names, you can also disable researching all together if you wish to allow players
+the ability to use all of Slimefun right off the bat.
+
+Any changes you make should be saved, then restart the server. Again, ***do not use /reload.*** If you are experiencing issues, and you issued a server reload,
+just stop and restart the server, since this fixes most issues.
+
+# Additional Addons
+
+If you wish to install additional addons, refer to [this](https://thebusybiscuit.github.io/builds/) page to see all the addons that are up to date on 1.14.4.
```
Comment (TheBusyBiscuit) [inline pages/installing-slimefun.md]: This page has nothing to do with Slimefun Addons.
You should refer to this page instead: https://github.com/TheBusyBiscuit/Slimefun4/wiki/Addons
(The link won't work yet, as the file is part of this repository and not yet part of the live wiki)

File: pages/installing-slimefun.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Installing Slimefun
```
Comment (TheBusyBiscuit) [inline pages/installing-slimefun.md]: You actually shouldn't add this header, the wiki automatically does it, so you will have two headers now.,

File: pages/installing-slimefun.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
```
Comment (TheBusyBiscuit) [inline pages/installing-slimefun.md]: BTW you should rename this file to "Installing-Slimefun", wiki files should use capital letters at the beginning of words.

File: pages/installing-slimefun.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+## Prerequisites
+* A 1.14.4 Spigot Minecraft server.
+* The latest version of Slimefun 4, which can be found [here](https://thebusybiscuit.github.io/builds/TheBusyBiscuit/Slimefun4/master/).
+* Slimefun 4 will utilize [CS-CoreLib](https://thebusybiscuit.github.io/builds/TheBusyBiscuit/CS-CoreLib/master/) as a mandatory dependency.
```
Comment (TheBusyBiscuit) [inline pages/installing-slimefun.md]: I think "utilize" is the wrong word here, you should maybe just outright tell them they will also need CS-CoreLib.

File: pages/installing-slimefun.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
+# Installing Slimefun
+
+## Prerequisites
+* A 1.14.4 Spigot Minecraft server.
```
Comment (TheBusyBiscuit) [inline pages/installing-slimefun.md]: Slimefun isn't made for Spigot, it is a Bukkit Plugin.

And hard-coding this version onto this page is not a good idea, you'd have to update it everytime a new Minecraft Version comes out.
Just tell them to use a Slimefun & CS-CoreLib Version that is compatible with their Version of Minecraft.

File: pages/installing-slimefun.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+* The latest version of Slimefun 4, which can be found [here](https://thebusybiscuit.github.io/builds/TheBusyBiscuit/Slimefun4/master/).
+* Slimefun 4 will utilize [CS-CoreLib](https://thebusybiscuit.github.io/builds/TheBusyBiscuit/CS-CoreLib/master/) as a mandatory dependency.
+
+
+## How to install
+
+Using Spigot, drag and drop the necessary jar files into your plugins directory. Then, restart your server.
```
Comment (TheBusyBiscuit) [inline pages/installing-slimefun.md]: Again, Spigot is too specific.
Slimefun is made for Bukkit.

File: pages/installing-slimefun.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+deleting it results in loss of ALL Slimefun related data, such as Levels, Unlocked Items, etc.
+
+## Configuring Slimefun
+
+This guide assumes you only have Slimefun 4 and CS-Corelib installed on your server.
+
+When viewing the Slimefun plugin folder, you will notice 5 different .YML files. Start by viewing *config.yml* in your favorite text editor.
+Personally, I recommend [Notepad++](https://notepad-plus-plus.org).
+
+Most of the things in this file are very self explanatory, from enabling certain items to choose how Slimefun Research behaves in creative mode.
```
Comment (TheBusyBiscuit) [inline pages/installing-slimefun.md]: Enabling items is done in the Items.yml which I don't even see listed here?

File: pages/installing-slimefun.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+Slimefun uses an Autoupdater to check for updates periodically, if you wish to disable this, or your host has somehow disabled things like that, set it to *false*
+
+Another important file is **whitelist.yml**, if you want to enable or disable certain items. If you install multiple addons for Slimefun, this file can get very big,
```
Comment (TheBusyBiscuit) [inline pages/installing-slimefun.md]: If you want to enable or disable certain items *in a specific world*.

**Comment by taydeo** [comment]
Updated the page including the changes you requested.

**Pull request merged**